packages/zowe-explorer-api/src/profiles/ProfilesCache.ts

```diff
@@ -78,15 +79,58 @@ export class ProfilesCache {
      *
      * @returns {imperative.ICommandProfileTypeConfiguration[]}
      */
-    public getConfigArray(): imperative.ICommandProfileTypeConfiguration[] {
+    public static getConfigArray(): imperative.ICommandProfileTypeConfiguration[] {
         return ProfilesCache.sessionProfileTypeConfigurations;
     }
 
+    /**
+     * Consider using the static ProfilesCache.getConfigArray() instead as `sessionProfileTypeConfigurations` (return value) is a static property.
```

Comment:
Should we deprecate the non-static version of this method, so that we don't have to maintain both versions?